lib/browser/growl.js

```diff
@@ -63,23 +63,29 @@ exports.notify = function(runner) {
  */
 function isPermitted() {
   var permitted = {
-    granted: Promise.resolve(true),
-    denied: Promise.resolve(false),
-    default: (function ask() {
-      Notification.requestPermission().then(function(permission) {
-        return Promise.resolve(permission === 'granted');
+    granted: function allow() {
+      return Promise.resolve(true);
```

Comment:
I totally spaced on noticing this, but you'll need to grab a promise shim for IE.  from what I can tell, [es6-promise](https://npm.im/es6-promise) looks ok

Reply:
That would go in the documentation, not the code. I don't have any means of testing old Windows browsers though.

Reply:
We support IE11 out-of-the-box and those tests run as job #11 on every build.  We don't require users of IE11 to pull in shims.

If job #11 passes--assuming there's coverage, which needs to live in `test/unit`--you're fine.

It's possible to launch these tests from a local dev environment--I'll message you on Gitter w/ the details.

Reply:
actually, you can probably avoid adding the shim if you can ensure we won't ever attempt to call `Promise.*`.  I think `isCapable()` is our escape hatch.  can you confirm?

Reply:
PR has passed all CI tests since originally pushed. If your browser doesn't support the feature (which IE doesn't), `isCapable` won't enable. If I can find someone brave enough to have it installed, I'll see if adding shims for _both_ promises and notifications will allow it to work.